packages/contracts/deploy/40_finalize-managing-dao/30_install-multisig-on-managing-dao.ts

```diff
@@ -80,6 +80,30 @@ const func: DeployFunction = async function (hre: EHRE) {
     `Prepared (Multisig: ${installationPreparedEvent.plugin}) to be applied on (ManagingDAO: ${managingDAOAddress}), see (tx: ${prepareTx.hash})`
   );
 
+  // Adding plugin to verify array
+  const multisigSetupAddress = await getContractAddress('MultisigSetup', hre);
+  const multisigSetup = await MultisigSetup__factory.connect(
```

Comment:
just small thing. `connect` doesn't need `await`. Doesn't vscode give you warning for it ?

Reply:
nope because it is still valid code. But I removed them now